src/components/errorPage/index.tsx

```diff
@@ -61,14 +62,33 @@ const ErrorPage: FC<Props> = ({ statusCode, language, onButtonClick }) => {
                       {t(`errorPage.${statusCode}.description`)}
                     </Text>
                   </Stack>
-                  <Button
-                    href={`/${language}`}
-                    as="a"
-                    onClick={onButtonClick}
-                    variant="secondary"
+                  <SimpleGrid
+                    columns={{
+                      base: 1,
+                      sm: showReloadPage ? 2 : 1,
+                    }}
+                    alignItems="center"
+                    spacing="md"
                   >
-                    {t(`errorPage.${statusCode}.buttonLabel`)}
-                  </Button>
+                    {showReloadPage ? (
+                      <Button
+                        onClick={() => {
+                          window.location.reload();
+                        }}
+                      >
+                        {t('errorPage.reloadPage')}
+                      </Button>
+                    ) : null}
+                    <Button
+                      onClick={() => {
+                        onButtonClick?.();
+                        window.location.href = `${window.location.origin}/${language}`;
```

Comment:
the idea here is to force a full page reload to reset the error boundary. With a href and relative link, nextjs will do a client side transition which doesn't reset the error state.